.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Shiny new `go-centrifuge` issue !!!
```

Comment:
Let's not add `Shiny` :)

Reply:
This is not something you should use, just a replaceable title.

Reply:
I will put a <replace> tag

Reply:
I would prefer maybe a simpler template at the current stage.

Are there any existing templates, we could simple re-use?

Reply:
Its the simplest I could think of, and its based on some existing templates I found on the internet :) 
`Simpler` is subjective, please make specific suggestions on how you see it can be simplified.